src/Bicep.LangServer/Completions/CompletionItemBuilder.cs

```diff
@@ -100,9 +100,12 @@ public static CompletionItem Preselect(this CompletionItem item, bool preselect)
             return item;
         }
 
-        public static CompletionItem WithCommand(this CompletionItem item, Command command)
+        public static CompletionItem WithCommand(this CompletionItem item, Command? command)
         {
-            item.Command = command;
+            if (command is not null)
```

Comment:
Not sure if the if-then is desirable.

Reply:
Reverted and cleaned up some code around this:
https://github.com/Azure/bicep/commit/fa1f6841fc84957acacb233e9c174576723be1e4